client/state/comments/reducer.js

```diff
@@ -133,6 +139,7 @@ export function requests( state = {}, action ) {
 
 	return state;
 }
+requests.schema = requestsSchema;
```

Comment:
For example, let's say we persist `COMMENTS_REQUEST`, but then we immediately reload the page. We have `COMMENTS_REQUEST` in state, but there's no in-flight request happening.